release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
-  * `is:open is:pr milestone:v1.14"` [milestone open PRs, no exclusions](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+milestone%3Av1.14)
+### Mid-Release Cycle
 
+This stage lasts from around week 6 until the code freeze starts (usually week 9). At this point, the Bug Triage team should start [escalating issues/PRs](#how-to-escalate) and communicating deadlines with contributors and SIG leads.
 
-### Reports [Before Freeze]
+There are three main tasks that should be done:
 
-At this point, release meetings start occurring ~3 times per week and a report is expected in each of them.
+- Ensure that issues and PRs targeting the release have a correct *milestone* and *priority* labels
+  - Linked issues and PRs should have the same `priority` and `kind` labels and the milestone
+- Ensure that `priority/critical-urgent` issues and PRs are not blocked
+  - If you find such `priority/critical-urgent` issue/PR you should ping the owner and owner SIGs, as well as, alert the release team leads.
+- Ping all issues and PRs to remind contributors and SIG leads that we are about one month away from the code freeze and what does it mean.
+  - You may want to ping issues at least two times to ensure deadlines are communicated correctly, usually once this stage starts and a couple of days before the code freeze starts.
 
-You should soon start maintaining an overall list of items/areas/issues which seem to be delaying, at risk of not making it in time, or in need of attention by the Release Lead or other roles, and present it in the meetings.
+At the beginning, you should prioritize pinging issues/PRs that haven't been updated for a longer time. You can use the following queries: [`is:issue is:open milestone:v1.17 sort:updated-asc`](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+milestone%3Av1.17+sort%3Aupdated-asc) and [`is:pr is:open milestone:v1.17 sort:updated-asc`](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+milestone%3Av1.17+sort%3Aupdated-asc).
 
-The Release Lead will include a template for this information in the Monday/Wednesday/Friday release team meeting notes.
+For pinging, the following template can be used:
 
-Code Freeze hasn't really started yet at this point, and it depends on the overall status and volume of open issues/PRs on how many to report. It's good to have some numbers of total issues/PRs available to see general trends over time, like increased or lowered volumes.
+```
+Hello! This <issue|PR> has not been updated for a long time, so I'd like to check what's the status. The code freeze is starting <insert date> (about <number of weeks> from now) and while there is still plenty of time, we want to ensure that each PR has a chance to be merged on time.
 
-### Issue Categorization
```

Comment:
I don't think we had Green/Yellow/Red reports for each issue in the past two cycles, but instead, we did this for in general.

Reply:
yes, I think clarification is good - we don't need the individual issue colors.